src/components/Topology/topology-card.js

```diff
@@ -124,20 +139,22 @@ export function TopologyCard({
           </div>
         ) : (
           <>
-            <div className="w-med-card-left py-4 pl-5 pr-1 overflow-auto">
-              {properties.map((property, index) => (
-                <Property
-                  key={property.name}
-                  property={property}
-                  className={
-                    index === _topology.properties.length - 1
-                      ? "mb-0"
-                      : "mb-2.5"
-                  }
-                />
-              ))}
-            </div>
-            <div className="w-med-card-right pl-1 py-4 pr-5 overflow-y-auto">
+            {Boolean(properties.length) && (
```

Comment:
I think this would show `false` in html, if `properties.length` is 0.

Reply:
JSX, won't render boolean values, they should be converted to strings to get them rendered. I think we are good here.